src/mcp_server_helm/commands/repository.py

```
@@ -0,0 +1,89 @@
+
+    if pass_credentials:
+        cmd.append("--pass-credentials")
+
+    return execute_helm_command(cmd)
+
+
+def helm_repo_remove(name: str) -> str:
+    """
+    Removes a chart repository.
+    """
+    logger.info(f"Running helm repo remove with name={name}")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)